scripts/tag-based-pre-processing/1-make_gene_matrix-alevin.R

```diff
@@ -11,14 +11,39 @@
 #        directory is specified, alevinQC is not run. (Optional)
 # "-l" - Optional label to add to output files. Generally necessary if processing
 #        multiple datasets in the same pipeline.
-#
+# "-f" - Optional but suggested filter application. If
+#        min_counts = 0.0001, num_genes = 3, num_genes = 50
+# "-r" - Store gene matrix as an RDS file instead of a tsv file. This is
+#        advisable for particularly large datasets.
+#        
+# Filter options:
+# Note: if any filter option is used, GeneMatrixFilter will be used for filtering,
+# and filter options not specified will be set to default. If you want all filters
+# to be automatically the default settings, use -f alone. 
+# 
+# "-f" - Option to use all default filters to be used as arguments to be used in 
```

Comment:
I find the repetition of this here confusing. How about moving some of the text here up above `Filter options`?

Reply:
Yeah, that was an accident.